src/tribol/utils/TestUtils.cpp

```diff
@@ -1174,6 +1174,10 @@ int TestMesh::tribolSetupAndUpdate( ContactMethod method,
                            enforcement,
                            BINNING_GRID );
 
+   std::cout << "Before tribol::setLoggingLevel()." << std::endl;
+   setLoggingLevel(csIndex, WARNING);
+   std::cout << "After tribol::setLoggingLevel()." << std::endl;
```

Comment:
Should we remove these `cout`s before merging? I assume they're only here for local debugging.